hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/ReplicationSource.java

```diff
@@ -87,9 +86,10 @@
   private static final Logger LOG = LoggerFactory.getLogger(ReplicationSource.class);
   // Queues of logs to process, entry in format of walGroupId->queue,
   // each presents a queue for one wal group
-  private Map<String, PriorityBlockingQueue<Path>> queues = new HashMap<>();
+  //private Map<String, PriorityBlockingQueue<Path>> queues = new HashMap<>();
```

Comment:
Remove rather than comment out (This is a draft so maybe you are just trying stuff -- if so, ignore this comment)